QUICK_FIX_SUMMARY.md

```
@@ -0,0 +1,123 @@
+### Solution
+
+**Update patterns to match normalized format:**
+
+```php
+// In SouthKorea.php, update:
+public const PATTERN_RRN = '^\d{13}$';  // 13 digits
+public const PATTERN_BRN = '^\d{10}$';  // 10 digits
+public const PATTERN = '^(\d{13}|\d{10})$';
+```
+
+**OR use the same validate() override approach as Switzerland**
```

markdownlint-cli2 (0.20.0)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)